docs/extending.md

```diff
@@ -0,0 +1,19 @@
+# Extending the adapter/serializer
+
+More than likely, you will need to add you own tweaks to the adapter adapter and (more often) the
+serializer.  EDA provides blueprints to make this easy.  For example, to make a customizable
+serializer for a User model:
+
+```console
+ember generate drf-serializer user
+```
+
+This will create app/serializers/user.js for you to customize.
+
+Similarly, if you wanted to, say, extend the adapter on the application level:
```

Comment:
`... to, say, extend the adapter ...`

Suggestion: I think it's clearer if you use 'for example' instead of 'say'. Or even just leave 'say' out entirely.